### ISSUE TITLE
No audio after 5-10 seconds playing video live stream in device

### ISSUE THREAD

**Issue by vminc** [yixia/Vitamio-iOS#3]
When I play a video stream, it works fine on simulator, but when I build on device, the following problem always happen:
- After video plays for 5-10 seconds, the audio suddenly turns off.

My device is iPhone5, running iOS 6.1.

**Comment by EchoLiao** [comment]
Pls submit your stream url for us to test or locate that problem..

**Comment by vminc** [comment]
The stream URL is http://112.197.2.11:1935/live/vtv3.stream/playlist.m3u8. Thanks!

**Comment by EchoLiao** [comment]
is this URL valid? we can't play it with any player even in PC player or browser.
But we can download the m3u8 file,  the m3u8 file may be invalid.

**Comment by EchoLiao** [comment]
I tested it again and can play now. it have problem with `VMDecodingSchemeSoftware` decoding scheme as you push above.

To solve this problem just change decoding scheme to `VMDecodingSchemeQuickTime`, 

```
- (void)mediaPlayer:(VMediaPlayer *)player setupManagerPreference:(id)arg
{
    player.decodingSchemeHint = VMDecodingSchemeQuickTime;
}
```

**Comment by vminc** [comment]
Thanks for your help!
Will this solution work for all video formats or it only works for quickTime format?

**Comment by EchoLiao** [comment]
All formats.  btw, You don't need worry about it, even if  vitamio failed with VMDecodingSchemeQuickTime, it also auto change to other scheme.

**Comment by vminc** [comment]
Thanks !

**Comment by vminc** [comment]
The problem still happens even if I use VMDecodingSchemeQuickTime.
Here's the stream I used to test rtmp://113.161.212.25/live/atv1

The audio only disappears if I don't pause/start the player during buffering :
- (void)mediaPlayer:(VMediaPlayer *)player bufferingStart:(id)arg
  {
  [player pause];
  }
- (void)mediaPlayer:(VMediaPlayer *)player bufferingEnd:(id)arg
  {
  [player start];
  }

However if I pause/start the player like above, the video player does not play as smooth as vitamio for Android under the same network condition. The buffer size I used is 200 000.
